can/interfaces/slcan.py

```diff
@@ -87,9 +94,7 @@ def __init__(self, channel, ttyBaudrate=115200, bitrate=None,
                                        bitrate=None, rtscts=False, **kwargs)
 
     def write(self, string):
```

Comment:
Shouldn't this be "private" (with a leading underscore)?

Reply:
Not necessarily. It could be useful to allow developers to send custom commands.